### PR TITLE
Features/contact us

### PR DESCRIPTION
**Pull request by Mughees-08**

### CODE REVIEW

File: src/app/Components/GenericComponents/image-grid/image-grid.component.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* Load More Button  */
+.btn-secondary {
```
Comment (SarahFurqan) [inline src/app/Components/GenericComponents/image-grid/image-grid.component.css:2]: Move btn-secondary to main css as its common in every comopnent

File: src/app/Components/GenericComponents/news-letter/news-letter.component.css [hunk 2]
```diff
@@ -14,28 +15,55 @@
...
+    
     width: 45%;
     margin-left: 100px;
     border-color: #000000;
-    
+
 }
 
-.btn-primary{
+.btn-primary {
```
Comment (SarahFurqan) [inline src/app/Components/GenericComponents/news-letter/news-letter.component.css:37]: move btn-primary to main css too

File: src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+}
+
+.row.mt-4 {
+    display: flex;
+    justify-content: center;
+}
+
+.btn-primary {
+    background-color: var(--secondary-color);
```
Comment (SarahFurqan) [inline src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css:37]: This primary button have secondary -color why?

File: src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    font-family: 'Outfit';
+    width: 13%;
+}
+
+.btn-primary:hover {
+    background-color: var(--primary-color);
+}
+
+.btn-secondary {
+    background-color: rgb(219, 217, 217);
```
Comment (SarahFurqan) [inline src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css]: color has not in variable

**Review by SarahFurqan — CHANGES_REQUESTED** [review]
Please btn classes to main class and primary and secondary color should be very clear in all css. Please resolve.

File: src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css (hunk 3 above)
Comment (Mughees-08) [reply src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css:37]: secondary color is basically a generic color. and it is being used all over the page.

File: src/app/Components/MainDashbordComponents/footer/footer.component.html [hunk 5]
```diff
@@ -124,7 +118,7 @@ <h5>
 <div class="container-fluid p-0">
     <div class="container">
         <p>Copyright 2023 © All Right Reserved Design by KS-Technologies</p>
-        <div class="end1">
+        <div class="text">
```
Comment (SarahFurqan) [inline src/app/Components/MainDashbordComponents/footer/footer.component.html:121]: text class is not a valid class please use appropriate naming conventions

File: src/app/Components/Pages/home/about/about.component.css [hunk 6]
```diff
@@ -13,11 +13,13 @@
     color: white;
     font-weight: 600;
     font-size: 44px;
-    font-family: Outfit;
+    font-family: 'Outfit';
```
Comment (SarahFurqan) [inline src/app/Components/Pages/home/about/about.component.css:16]: Font family is same on each component please create one class for font-family and add that class

File: src/app/Components/Pages/home/about/about.component.html [hunk 7]
```diff
@@ -20,7 +20,7 @@ <h3 class="name mt-4">
                 electronic typesetting, remaining essentially unchanged.
             </p>
             <a routerLink="about" routerLinkActive="active" ariaCurrentWhenActive="page" class="active-link">
-                <span style="color: #29416F; font-weight: 700; font-family: Outfit; ">Read More -></span>
+                <span style="color: #29416F; font-weight: 700; font-family: Outfit; ">Read More</span>
```
Comment (SarahFurqan) [inline src/app/Components/Pages/home/about/about.component.html:23]: please don't use inline style

File: src/app/Components/Pages/home/blog/blog.component.css [hunk 8]
```diff
@@ -1,39 +1,35 @@
...
-.h2 {
-    font-family: outfit;
+.card-title {
+    color: #5EBE80;
+    font-family: 'outfit';
 }
 
-.p {
-    font-family: outfit;
+.text-muted {
```
Comment (SarahFurqan) [inline src/app/Components/Pages/home/blog/blog.component.css:22]: inappropriate class name

File: src/app/Components/Pages/home/landing-page/landing-page.component.html [hunk 9]
```diff
@@ -2,9 +2,8 @@
 
     <div class="row">
         <div class="row text-center heading">
-            <h1><span style="color: var(--primary-color);">KS-Technologies</span><span> is a Perfect</span>
-                <br><span>Solutions
-                    For your Business.</span>
+            <h1><span style="color: var(--primary-color);">KS-Technologies</span> is a Perfect
```
Comment (SarahFurqan) [inline src/app/Components/Pages/home/landing-page/landing-page.component.html:5]: remove inline style

**Review by SarahFurqan — CHANGES_REQUESTED** [review]

**Pull request merged**